test/lit.cfg

```diff
@@ -1891,9 +1891,11 @@ elif run_os == 'linux-androideabi' or run_os == 'linux-android':
     config.target_pic_opt = "-fPIC"
 
     ndk_platform_tuple = get_architecture_value(armv7="armeabi-v7a",
-                                                aarch64="arm64-v8a")
+                                                aarch64="arm64-v8a",
+                                                x86_64="x86_64")
     ndk_platform_triple = get_architecture_value(armv7="arm-linux-androideabi",
-                                                 aarch64="aarch64-linux-android")
+                                                 aarch64="aarch64-linux-android",
+                                                 x86_64="x86_64-linux-android")
```

Comment:
Can you just remove these `ndk_*`variables and the prior `get_architecture_value` method instead? They are entirely unused, as raised by #63907.

Reply:
Never mind, I removed it myself, #79263.